### ISSUE TITLE
Trimola spawn egg not spawning anything.

### ISSUE THREAD

**Issue by starspunwolf** [MsRandom/BeastsMod#22]
I tested spawning the new creatures in the latest release of Beasts. Slime Slug and Butterfly fish spawn just fine, but Trimolas do not. Using a spawn egg doesn't spawn anything at all, nor does it consume the spawn egg item.

I also double-checked spawn eggs for all of the old creatures, and they all work fine.

I tested this on both my server and in singleplayer; the failure to spawn occurs in both.

Full mod list:
https://pastebin.com/KjJWbAhw

**Comment by Coda1552** [comment]
Can you please provide the console log after trying to spawn a Trimola?

**Comment by starspunwolf** [comment]
Certainly! Apologies that I didn't think to do so:
https://pastebin.com/B4yF6nE2

Also, a short video showing a handful of attempts (doubtful this is helpful, but I figured it can't hurt to provide it):
https://i.gyazo.com/33bed215b6bbc968fcbd35ee7e03f936.mp4

**Comment by Coda1552** [comment]
Thanks, we'll take a look at it!

**Comment by Coda1552** [comment]
Try reinstalling the mod and let us know if it works. Thanks!

**Comment by starspunwolf** [comment]
I updated to the latest version of Beasts and completely reset my world (I also removed a few other mods, namely: OTG, VanillaVistas, and Shamanism). Unfortunately, even with the newest version of Beasts newly installed, the issue with the trimola spawn egg persists. 

Thank you for looking into it, btw! Your mod is wonderful and I appreciate all your hard work.

**Comment by starspunwolf** [comment]
Addendum: I'm not certain if this is directly related or not, but I thought I'd include it here rather than create a new issue. If you'd like me to post this to its own issue, I'm happy to do so.

Upon attempting to explore the aforementioned newly-created world, anytime I attempt to move within/near the Dried Reef biome, the server crashes. Doing a quick glance at the crash log, it appears to be related to the trimola.

https://pastebin.com/tBMNphUQ

**Comment by Coda1552** [comment]
Gosh, this is so weird. We will look into it a bit more

**Comment by starspunwolf** [comment]
Update with new information: As of the newest version of Beasts, I have the spawn egg issue with landwhales as well. I apologize for not checking this sooner after installing the newest version. It's identical to the trimola spawn egg issue. However, I continued to be able to spawn the other (non-landwhale, non-trimola) creatures just fine. Just adding this in case it's helpful information!

**Comment by MsRandom** [comment]
I'll fix it as soon as possible.

**Comment by MsRandom** [comment]
It seems to be a problem with other mods since it's not happening on my end, can you test if this happens with Beasts alone without any other mods?

**Comment by starspunwolf** [comment]
You're right! I did as asked and created a world with Beasts as the only mod. Both spawn eggs work. I also flew around until I found a Dried Reef biome, and there was no crash. I can only test this in singleplayer, as I have a modpack running for my players on multiplayer, but I think it's safe to say it's another mod. I'll be sure to test each one individually with Beasts.

Thank you!

**Comment by Coda1552** [comment]
Awesome, I'm glad it worked. Keep us updated if you figure out what mod it is, please. 👍

**Comment by starspunwolf** [comment]
After testing each mod one by one, I wasn't able to reproduce either issue (spawn eggs or dried reef crash). So, I thought I'd create another singleplayer test world with the entire modpack, and the issue didn't exist there either (whereas it did exist in singleplayer previously, when I first started this issue thread).

The issue now seems to only exist in multiplayer. Unfortunately, I can't scrap the multiplayer world and test each mod on its own in multiplayer for the aforementioned reason. I can say though that the current multiplayer world is brand new as of the latest Beasts release. So strange!

**Comment by starspunwolf** [comment]
Addendum: The errors for both the spawn egg issue and the dried reef crash seem to reference something pertaining to keybindings and server side? I checked my keybindings and didn't see anything pertaining to this, though I'd be surprised if it had anything to do with client-side settings since the issue only exists in multiplayer/server.

Sorry for the extra message, just occurred to me to mention!

See these two pastebins:
https://github.com/MsRandom/BeastsMod/issues/22#issuecomment-583806527
https://github.com/MsRandom/BeastsMod/issues/22#issuecomment-585880257

**Comment by Coda1552** [comment]
Hello, I have tested Beasts on the server and it is indeed a Beasts problem, it will be fixed in the next update, most likely this week. Thank you for reporting this issue :)

**Comment by starspunwolf** [comment]
Thank you so much!
